src/index.js

```diff
@@ -0,0 +1,33 @@
+// eslint-disable-next-line no-unused-vars
+import _ from 'lodash';
+import './style.css';
+
+const todoLists = [
+  {
+    description: 'Working on javascript todo app',
+    completed: false,
+    index: 1,
+  },
+  {
+    description: 'Work on ruby on rails',
+    completed: true,
+    index: 2,
+  },
+  {
+    description: 'Work on php laravel',
+    completed: true,
+    index: 8,
+  },
+  {
+    description: 'Work on java',
+    completed: false,
+    index: 6,
```

Comment:
- [ ] As per project requirements on page load, the task array is rendered dynamically to the DOM. The tasks should appear in order of the index values for each task. Here the `list with index 8` should come after the `list with index 6`. But it is not the case in your project. Please try to fix this issue.

Hint:- We can loop over the list items and append the list in DOM as per the index value.
![image](https://user-images.githubusercontent.com/53019916/126498505-8e33d18a-1640-4851-b225-48b8de7d0bb8.png)